### DISCUSSION TITLE
MCD43

### DISCUSSION THREAD

**Discussion by jgomezdans** [microsoft/PlanetaryComputer#143] [Data request]
Hi, it'd be very good to be able to use the SIAC atmospheric correction scheme (https://github.com/MarcYin/SIAC, [description paper](https://gmd.copernicus.org/articles/15/7933/2022/)) to run on the PC. However, this requires access to Landsat and S2 Level 1 products (see #5 ;-D), but also access to the [MCD43A1 product](https://lpdaac.usgs.gov/products/mcd43a1v061/). It can be downloaded and stored in blobs, but obviously, it'd be much better to have it online.

**Comment by jgomezdans** [comment]
Hello, is there any plan to get the MCD43 product(s) available? If not, is there  a "preferred way" of accessing these kind of data from PC?

**Reply by TomAugspurger** [reply]
No plans right now, but we can add it to the backlog.
